paymentsheet/src/main/java/com/stripe/android/lpmfoundations/luxe/TransformSpecToElements.kt

```diff
@@ -77,7 +77,7 @@ internal class TransformSpecToElements(
                     arguments.shippingValues
                 )
                 is SepaMandateTextSpec -> it.transform(arguments.merchantName)
-                is PlaceholderSpec -> error("Placeholders should be processed before calling transform.")
+                is PlaceholderSpec -> null // Placeholders should be processed before calling transform.
```

Comment:
Unrelated, happy to break into a separate PR if you want.

Reply:
should this be logged as an unexpected error?

Reply:
I don't think so, we have tests to validate this behavior.